pkgs/by-name/ty/typescript-go/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
```

Comment:
```suggestion
```

Reply:
I'd prefer to stick with the current formatting unless nixfmt-rfc-style indicates an error.

This test fixture seems has same sepration line: https://github.com/NixOS/nixfmt/tree/89598649a2a04e3c3c48d7d6b83e745a6e0b3e94/test/diff/idioms_pkgs_2